tempesta_fw/http.c

```diff
@@ -3884,8 +3885,8 @@ tfw_http_resp_process(TfwConn *conn, const TfwFsmData *data)
 /**
  * @return status (application logic decision) of the message processing.
  */
-static int
-__tfw_http_msg_process(void *conn, TfwFsmData *data)
+int
+tfw_http_msg_process_generic(void *conn, TfwFsmData *data)
```

Comment:
Bad naming. It's not generic. It's HTTP1 processing.

Reply:
In nearest future PRs (in context of #309) this function is planned for usage for HTTP/2 processing too, so I ran a little forward here. Most part of `tfw_http_msg_process_generic()` is general processing, whatever HTTP/1.1 or HTTP/2 protocols (except the parsing stage only), so it is worth to try adopt it for both protocols. In any case if this idea will fail, and the separate procedures will be needed - the name will be reverted accordingly.